ciri/api/tests/test_check_consistency.c

```diff
@@ -193,10 +197,18 @@ void test_check_consistency_false(void) { test_check_consistency_consistent_ledg
 
 void test_check_consistency_true(void) { test_check_consistency_consistent_ledger(true); }
 
-int main(void) {
+int main(int argc, char *argv[]) {
   UNITY_BEGIN();
   TEST_ASSERT(storage_init() == RC_OK);
 
+  if (argc >= 2) {
+    debug_mode = true;
+  }
+  if (debug_mode) {
+    test_db_path = "test.db";
+    ciri_db_path = "ciri.db";
+  }
+
```

Comment:
Why ?

Reply:
Forgot to remove, debugging purposes though

Reply:
outdated